mono/mini/interp/transform.c

```diff
@@ -4321,6 +4321,12 @@ generate (MonoMethod *method, MonoMethodHeader *header, InterpMethod *rtm, unsig
 		case MONO_CUSTOM_PREFIX:
 			++td->ip;
 		        switch (*td->ip) {
+				case CEE_MONO_RETHROW:
+					CHECK_STACK (td, 1);
+					SIMPLE_OP (td, MINT_THROW);
```

Comment:
`MINT_MONO_RETHROW`

Reply:
Hmm, it seemed to be working locally. Let me make sure this is being run during the tests.

Reply:
It was being used but apparently MINT_THROW doesn't break the Invoke stack trace. Not sure if that's an interpreter bug.